added wufoo service provider and feed parser

jerome-poisson opened the pull request:
Wufoo is form building service, this commit add feeding service and
feed parser using its API.

this also fix a bug in superdesk/io/registry.py forbidding using a
feeding service and a feed parser with the same name.

It also add a last_ingested_id field for feeding services which can be
used to only ingest new items, this is more resilient than using a date.

Fixed documentation for register_feeding_service and register_feed_parser

SDNTB-377

ioanpocol commented:
There are a few NTB related hardcoded values and I think we have to move it on NTB repo or to add additional settings in order to map those values to current vocabulary.

petrjasek commented:
this should be added to docs, there is a list of parsers/services

petrjasek commented:
and tests pls

jerome-poisson commented:
@ioanpocol actually the whole parser is related to NTB, it make more sense to move it there indeed, I can keep the feeding service here.

@petrjasek didn't knew parser were in docs, will add it. And for tests the thing is that it's external service, but I can still add unit testing with mocks.

thanks for reviewing.

petrjasek commented:
sure, parser tests should be enough

jerome-poisson commented:
@ioanpocol I've moved the parser to https://github.com/superdesk/superdesk-ntb/pull/99
@petrjasek I've added (and fixed) doc for feedind_service. I've also added the test which is in the above-mentioned NTB P.R.

jerome-poisson commented:
@ioanpocol @petrjasek can you validate the review ? I need it to merge this P.R. (according to @karelpetrak I can merge now)
